linkml/generators/docgen/slot.md.jinja2

```diff
@@ -20,9 +20,35 @@ URI: [{{ gen.uri(element) }}]({{ gen.uri(element) }})
 <!-- no inheritance hierarchy -->
 {% endif %}
 
+{% if schemaview.is_mixin(element.name) %}
+## Mixin Usage
+
+| mixed into | description | range | domain |
+| --- | --- | --- | --- |
+{% for s in schemaview.slot_children(element.name) -%}
+| {{ gen.link(s) }} | {{ schemaview.get_slot(s).description }} | {{ schemaview.get_slot(s).range }} | {{ schemaview.get_classes_by_slot(schemaview.get_slot(s))|join(', ') }} |
+{% endfor %}
+{% endif %}
+
 ## Properties
 
- * Range: {{gen.link(element.range)}}
+* Range: {{gen.link(element.range)}}
+* Multivalued: {{ element.multivalued }}
+{% if element.aliases %}
+* Aliases:
+    {% for alias in element.aliases -%}
+        * {{ alias }}
```

Comment:
is the indentation correct?

Reply:
Seems like it: https://microbiomedata.github.io/nmdc-schema/dissolution_reagent/?

Reply:
But I'll correct it anyway